dist/PublicLab.Editor.css

```diff
@@ -200,6 +202,337 @@ https://github.com/sliptree/bootstrap-tokenfield/pull/287 */
 .tooltip.bottom .tooltip-arrow {
   display:none;
 }
+
+#fusion
+{
```

Comment:
Please follow the convention and place the `{`on the same line with selector, not on the new line.
You should do the same for other lines.